functions/fundle.fish

```diff
@@ -380,6 +385,7 @@ function __fundle_plugin -d "add plugin to fundle" -a name
 		end
 	end
 	test -z "$plugin_url"; and set plugin_url (__fundle_get_url $name)
+    set name (echo "$name" | cut -d'@' -f1)
```

Comment:
I use spaces for indentation and didn't notice you were using tabs.
I'll fix that and squash the commits.